src/libUtils/SysCommand.h

```diff
@@ -17,17 +17,47 @@
 #ifndef __SYSCOMMAND_H__
 #define __SYSCOMMAND_H__
 
+#include <array>
 #include <cstdlib>
 #include <fstream>
 #include <iostream>
+#include <memory>
 #include <string>
 
-void ExecuteCmd(const std::string cmd)
+const std::string EXEC_CMD_LOG = "ExecuteCmd.txt";
+
+class SysCommand
 {
-    int ret = std::system((cmd + " > ExecuteCmd.txt").c_str());
-    (void)ret;
-    LOG_GENERAL(INFO,
-                "ExecuteCmd: " << std::ifstream("ExecuteCmd.txt").rdbuf());
-}
+public:
+    static void ExecuteCmd(const std::string cmd)
+    {
+        int ret = std::system((cmd + " > " + EXEC_CMD_LOG).c_str());
+        (void)ret;
+    }
+
+    static const std::string ExecuteCmdWithOutput(const std::string cmd)
```

Comment:
This function seems not ready for a pr: (1) not necessarily introduce exceptions (2) the memory resource (the file descriptor) is not handled properly when everything goes well (without triggering exception)